src/pages/ConnectWallet/MobileConnect.tsx

```diff
@@ -198,19 +209,23 @@ export const MobileConnect = () => {
         <BodyStack>
           <Stack textAlign='center' spacing={2}>
             <Heading>{translate('connectWalletPage.welcomeBack')}</Heading>
-            <BodyText>{translate('connectWalletPage.mobileSelectBody')}</BodyText>
+            {!isWaitingForRedirection && !state.isLoadingLocalWallet ? (
+              <BodyText>{translate('connectWalletPage.mobileSelectBody')}</BodyText>
+            ) : null}
```

Comment:
We don't want to say "Select a wallet in the list" if we are currently waiting for a wallet to be loaded